presentation/src/main/java/com/threegap/bitnagil/presentation/onboarding/component/atom/textbutton/TextButton.kt

```diff
@@ -0,0 +1,75 @@
+package com.threegap.bitnagil.presentation.onboarding.component.atom.textbutton
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsPressedAsState
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun TextButton(
+    modifier: Modifier = Modifier,
+    text: String,
+    onClick: () -> Unit,
+    enabled: Boolean = true,
+    defaultBackgroundColor: Color = Color(0xFF1A237E),
+    pressedBackgroundColor: Color = Color(0xFF0D123F),
+    disabledBackgroundColor: Color = Color(0xFFF0F0F0),
+    defaultTextColor: Color = Color.White,
+    pressedTextColor: Color = Color(0xFFA0A0A0),
+    disabledTextColor: Color = Color(0xFFB0B0B0),
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+    val isPressed by interactionSource.collectIsPressedAsState()
+
+    val backgroundColor = when {
+        !enabled -> disabledBackgroundColor
+        isPressed -> pressedBackgroundColor
+        else -> defaultBackgroundColor
+    }
+
+    val textColor = when {
+        !enabled -> disabledTextColor
+        isPressed -> pressedTextColor
+        else -> defaultTextColor
+    }
+
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(56.dp)
+            .clip(RoundedCornerShape(12.dp))
+            .background(backgroundColor)
```

Comment:
[2] tmi긴 한데 이렇게도 가능할거 같습니담.
```suggestion
            .background(
                color = backgroundColor,
                shape = RoundedCornerShape(12.dp),
            )
```